src/psyclone/tests/psyir/nodes/omp_directives_test.py

```diff
@@ -336,18 +330,22 @@ def test_omp_single_gencode(nowait):
         create(invoke_info)
     schedule = psy.invokes.invoke_list[0].schedule
 
-    _, _ = single.apply(schedule.children[0], {"nowait": nowait})
+    single.apply(schedule.children[0], {"nowait": nowait})
     omp_single_loop = schedule.children[0]
-    _,_ = parallel.apply(schedule.children[0])
+    parallel.apply(schedule.children[0])
     goceantrans = GOceanExtractTrans()
-    _,_ = goceantrans.apply(schedule.children[0])
+    goceantrans.apply(schedule.children[0])
 
     code = str(psy.gen)
+    print(code)
     string = ""
     if nowait:
         string = " nowait"
     assert (
-        "!$omp single{0}\n".format(string) in code)
+         "    !$omp parallel default(shared), private(i,j)\n" +
```

Comment:
This python seems to have an indent of 5 spaces rather than 4. Also it looks as though the generated `!$omp parallel` is not being correctly indented but that's not your problem.

Reply:
Done